### ISSUE TITLE
Saved the trained agent and hold the thetas unchanged for simulation on new dataset

### ISSUE THREAD

**Issue by hanchenresearch** [Nth-iteration-labs/contextual#18]
Dear Robin,

This is not a bug report but more like a new feature request.

We know that the theta is updated after the agents' every interaction with the bandit. What I want to ask is that is it possible to save the "trained" agent with the theta for later use on another dataset. The logic behind this is that the trained agent acts as an oracle/ground truth of the environment, then I want to add a benchmark full information model based on this oracle.In this way, I can look at what is the maximum reward I can theoretically get if I initiate my offline evaluation with this oracle, without knowing the ground truth until the ends of my simulation. 

Basically, to achieve this goal, I need to save the trained agents with the thetas, and break the thata updating chain and hold the thetas unchanged when used for another dataset.

Thank you so much for your help!

Best,
Han
